covid_graphs/covid_graphs/predictions.py

```diff
@@ -0,0 +1,101 @@
+from dataclasses import dataclass
+import datetime
+from typing import List
+
+from .formula import ATG_formula, Formula
+
+
+# Make the class hashable.
+@dataclass(frozen=True, eq=True)
+class PredictionEvent:
+    name: str
+    date: datetime.date
+
+
+@dataclass
+class CountryPrediction:
+    prediction_event: PredictionEvent
+    country: str
+    formula: Formula
+
+
+BK_20200329 = PredictionEvent(name="bk_20200329", date=datetime.date(2020, 3, 29))
+BK_20200413 = PredictionEvent(name="bk_20200413", date=datetime.date(2020, 4, 13))
+OTHER = PredictionEvent(name="other", date=datetime.date(2020, 4, 13)) # TODO: When did we add SVK prediction?
```

Comment:
This is not a prediction, but rather just trying to fit a polynomial on Slovak data . I think you can remove it from here and we should rather have it in the "fitting" script, when he have one.

Reply:
It is your prediction :)